package.json

```diff
@@ -10,7 +10,9 @@
   "author": "128 Technology",
   "license": "MIT",
   "private": true,
-  "files": ["index.js"],
+  "files": [
+    "index.js"
```

Comment:
Did prettier format this or NPM install?  Unfortunately npm install will autoformat these files and the formatting is a bit different than prettier which causes the file to appear changed

Reply:
npm i did it - it seems like prettier isnt kicking in the entire time I worked on this...not sure why

Reply:
Ah okay, you can leave it as NPM formatted it, we must have let prettier format it at some point